qemu/tests/blockdev_commit_backing_file.py

```diff
@@ -17,7 +19,7 @@ def check_backing_file(self):
         image_obj = qemu_storage.QemuImg(device_params,
                                          data_dir.get_data_dir(), device)
         output = image_obj.info()
-        logging.info(output)
+        LOG_JOB.info(output)
```

Comment:
Here is also safe to replace with `self.test` since its `test` is passed from `run(test, )`.